src/draive/conversation/realtime/default.py

```
@@ -1,210 +1,156 @@
-from draive.lmm.types import LMMContextElement
-from draive.multimodal import MetaContent
+from draive.multimodal import MultimodalContent
 from draive.tools import Toolbox
-from draive.utils import Processing, ProcessingEvent
 
-__all__ = ("realtime_conversation",)
+__all__ = ("realtime_conversation_preparing",)
 
 
-async def realtime_conversation(  # noqa: C901, PLR0915
+async def realtime_conversation_preparing(  # noqa: C901
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)